docs/intro/docker.md

```diff
@@ -0,0 +1,75 @@
+---
+order: 3
+description: Run Starport CLI using a Docker container.
+---
+
+# Run Starport in Docker
+
+You can run Starport CLI inside a Docker container using a Docker CLI without installing the Starport binary directly on your machine.
+
+Running Starport in Docker can be useful for various reasons; isolating your test environment, running Starport on an unsupported operating system, or experimenting with a different version of Starport CLI without installing it.
+
+Docker containers are like virtual machines because they provide an isolated environment to programs that runs inside them. In this case, you can run Starport in an isolated environment.
+
+Experimentation and file system impact is limited to the Docker instance. The host machine is not impacted by changes to the container.
+
+## Prerequisites
+
+Docker must be installed. See [Get Started with Docker](https://www.docker.com/get-started).
+
+## Scaffolding a Chain
+
+When Docker is installed, you can build a blockchain with a single command.
+
+To scaffold a blockchain `planet` in a new directory, run this command in a terminal window:
+
+```
+docker run -ti -v $HOME/sdh:/home/tendermint -v $PWD:/apps starport/cli:develop app github.com/hello/planet
+```
+
+Be patient, this command takes a minute or two to run because it does everything for you:
+
+- Creates a container that runs from the `starport/cli` image.
+- Executes the Starport binary inside the image. In this example, the `starport/cli:develop` image is on the `develop` branch so you can experiment with the next version.
+- Uses `-v $HOME/sdh:/home/tendermint` to map the `$HOME/sdh` directory in your local computer (the host machine) to the `/home/tendermint` directory inside the container.
```

Comment:
```suggestion
- Uses `-v $HOME/sdh:/home/tendermint` to map the `$HOME/sdh` directory in your local computer (the host machine) to the home directory (`/home/tendermint`) inside the container. Because Starport itself and chains you serve with Starport will persist some files.
```

Reply:
We should also document `-v $PWD:/apps` part, it'll persist the scaffold app in the container to the host machine at current working dir.